web/lib/src/helpers/extensions.dart

```diff
@@ -122,9 +123,8 @@ extension XMLHttpRequestGlue on XMLHttpRequest {
     // from Closure's goog.net.Xhrio.getResponseHeaders.
     final headers = <String, String>{};
     final headersString = getAllResponseHeaders();
-    final headersList =
-        LineSplitter.split(headersString).where((header) => header.isNotEmpty);
-    for (final header in headersList) {
+    for (final header in LineSplitter.split(headersString)
```

Comment:
I HATE iterables just hanging out in variables. Inlining it!

Reply:
And it wasn't a `List` 😄

Reply:
Ack - it's not even a list :D